### PR TITLE
fix(tabs): bordered and layout center combination works

### PR DESCRIPTION
**Pull request by Elijbet**
**Related Issue:** #5345

The component did not previously support bordered + centered combination. With this change `tabs` with `bordered layout="center"` no longer ignore the `bordered` attribute.

Addresses the concern that it may be unexpected that those combined props do not display the intended design (#4714, #3682)

https://user-images.githubusercontent.com/19231036/198109865-1aa5c379-0e95-40d9-8e98-b07bb324fb9f.mov

### CODE REVIEW

File: src/components/tabs/tabs.stories.ts [hunk 1]
```diff
@@ -70,7 +70,7 @@ simpleDarkThemeRTL_TestOnly.parameters = { themes: themesDarkDefault };
 
 export const bordered = (): string => html`
   <calcite-tabs
-    layout="inline"
+    layout="${select("layout", ["inline", "center"], "inline")}"
```
Comment (driskull) [inline src/components/tabs/tabs.stories.ts:73]: Should we add a visual screenshot for when the layout is "center" to capture the bordered state?

**Review by driskull — APPROVED** [review]
👍 

(ﾉ◕ヮ◕)ﾉ*:・ﾟ✧

**Comment by ashetland** [comment]
Looks great to me!

**Comment by Elijbet** [comment]
A follow-up question on this issue is whether there should be a border to the left/right of the container. 
MicrosoftTeams-image (2)

@ashetland @SkyeSeitz @macandcheese

**Comment by ashetland** [comment]
> A follow-up question on this issue is whether there should be a border to the left/right of the container. MicrosoftTeams-image (2)
> 
> @ashetland @SkyeSeitz @macandcheese

I would say no. This looks like the correct behavior.

**Review by ashetland — APPROVED** [review]

File: src/components/tabs/tabs.stories.ts [hunk 2]
```diff
@@ -214,14 +214,20 @@ export const disabledTabs_TestOnly = (): string => {
...
+      <calcite-tab-title>Tab 2 Title</calcite-tab-title>
+      <calcite-tab-title>Tab 3 Title</calcite-tab-title>
+      <calcite-tab-title>Tab 4 Title</calcite-tab-title>
+    </calcite-tab-nav>
+  </calcite-tabs>`,
+  createSteps("calcite-tabs")
+    .snapshot("Center Layout.")
+    .executeScript(`document.querySelector("calcite-tabs").bordered = "true"`)
+    .snapshot("Bordered state when layout is center.")
+);
```
Comment (Elijbet) [inline src/components/tabs/tabs.stories.ts:231]: I know we’re conscious of our screenshot count, but I got an ok from Ben with the current state. 😛

**Pull request merged**